include/seqan3/alphabet/adaptation/char.hpp

```diff
@@ -50,21 +50,10 @@
  * their type ambiguity with `int8_t` and `uint8_t`.
  *   * Note that `wchar_t` is absent from the list for its notorious brokenness (different sizes and signedness
  * between platforms); use `char16_t` or `char32_t` instead.
- *
- * \attention
- * Please be aware that if you also include `alphabet/concept.hpp` and/or `alphabet/adaptation/concept.hpp`,
- * you need to do so **after** including this file, not before.
  */
 
 #pragma once
 
-//!\cond
-#ifdef SEQAN3_ALPHABET_CONCEPT_INCLUDED
-#error You must include alphabet/concept.hpp and/or alphabet/adaptations/concept.hpp AFTER including \
-       alphabet/adaptation/char.hpp, not before!
```

Comment:
It's a little more complicated, but now you always get the adaptation alphabets when you include the alphabet concept. It's the only way to avoid the include order hassle. But otherwise this is entirely unrelated to the rest of the PR.